crates/ruff_linter/src/rules/ruff/mod.rs

```diff
@@ -384,6 +384,10 @@ mod tests {
     }
 
     #[test_case(Rule::ZipInsteadOfPairwise, Path::new("RUF007.py"))]
+    #[test_case(Rule::UnformattedSpecialComment, Path::new("RUF104.py"))]
+    #[test_case(Rule::UnformattedSpecialComment, Path::new("RUF104_1.py"))]
+    #[test_case(Rule::UnformattedSpecialComment, Path::new("RUF104_2.py"))]
+    #[test_case(Rule::UnformattedSpecialComment, Path::new("RUF104_3.py"))]
```

Comment:
Are these files testing different aspects? If so, can we give them more meaningful names than `_1` etc.?